src/modules/rc_update/rc_update.cpp

```diff
@@ -636,6 +638,11 @@ void RCUpdate::UpdateManualSwitches(const hrt_abstime &timestamp_sample)
 	switches.transition_switch = get_rc_sw2pos_position(rc_channels_s::FUNCTION_TRANSITION, _param_rc_trans_th.get());
 	switches.gear_switch       = get_rc_sw2pos_position(rc_channels_s::FUNCTION_GEAR,       _param_rc_gear_th.get());
 
+#if defined(ATL_MANTIS_RC_INPUT_HACKS)
+	switches.photo_switch = get_rc_sw2pos_position(rc_channels_s::FUNCTION_AUX_3, 0.5f);
```

Comment:
Can we make this configurable through the regular mechanisms and not carry the define? There will be other cases where it's useful to configure these switches.

Reply:
I would say yes but on the other hand we were trying to get away from these "one purpose only switches" towards the flight mode slot, and then use mavlink commands for the rest of it, as much as possible.

So, yes we could add switches for things like that but it won't stop there and we'll have to have many of them.

Reply:
Alright, I just want to be careful not to let per vehicle specifics creep into the code everywhere. So I guess we can live with it for now.